main/src/main/java/org/mobilitydata/gtfsvalidator/cli/Main.java

```diff
@@ -54,8 +62,21 @@ public static void main(String[] argv) {
         NoticeContainer noticeContainer = new NoticeContainer();
         GtfsFeedContainer feedContainer;
         try {
-            feedContainer = feedLoader.loadAndValidate(GtfsInput.createFromPath(args.input), feedName, validatorLoader,
-                    noticeContainer);
+            if (args.input == null) {
+                feedContainer = feedLoader.loadAndValidate(
+                        GtfsInput.createFromUrl(
+                                new URL(args.url),
+                                args.storageDirectory),
+                        feedName,
+                        validatorLoader,
+                        noticeContainer);
+            } else {
+                feedContainer = feedLoader.loadAndValidate(
+                        GtfsInput.createFromPath(args.input),
+                        feedName,
+                        validatorLoader,
+                        noticeContainer);
+            }
```

Comment:
Let's write it shorter.

GtfsInput gtfsInput = args.input == null ? GtfsInput.createFromUrl(
                                new URL(args.url),
                                args.storageDirectory)
  : GtfsInput.createFromPath(args.input);

feedContainer = feedLoader.loadAndValidate(
                        gtfsInput,
                        feedName,
                        validatorLoader,
                        noticeContainer);

Reply:
I'm not sure about our stance on the ternary operator. @LeoFrachet ?

Reply:
```suggestion
            GtfsInput gtfsInput = args.input == null ? GtfsInput.createFromUrl(
                                new URL(args.url),
                                args.storageDirectory)
  : GtfsInput.createFromPath(args.input);

feedContainer = feedLoader.loadAndValidate(
                        gtfsInput,
                        feedName,
                        validatorLoader,
                        noticeContainer);
```

Reply:
When I was software developer, I was open to use the ternary operator only if it fits in one line of code, aka if you could see the whole thing with just one glance, and we weren't allowing it when it was spanning on multiple lines.

Reply:
Thanks @LeoFrachet!

Reply:
I would agree with @LeoFrachet on that one. For readability, accessibility and maintenance.

Reply:
I think that there was a misunderstanding. My point was to avoid the code duplication, not to necessarily introduce ternary operation. If you do not like ?:, you can write it this way:

```
GtfsInput gtfsInput;
if  (args.input == null) {
   gtfsInput = GtfsInput.createFromUrl(
                                new URL(args.url),
                                args.storageDirectory);
} else {
 gtfsInput =  GtfsInput.createFromPath(args.input);
}
feedContainer = feedLoader.loadAndValidate(
                        gtfsInput,
                        feedName,
                        validatorLoader,
                        noticeContainer);

Reply:
This comment was not resolved :(

Reply:
Indeed, thanks @aababilov for flagging this. I open an issue so that we keep track of that and modify.